src/irr/video/COpenGLFunctionTable.h

```diff
@@ -40,3733 +15,3084 @@ wywali
+			,glEnableVertexArrayAttrib
+			,glEnableVertexArrayAttribEXT
+			,glDisableVertexAttribArray
+			,glDisableVertexArrayAttrib
+			,glDisableVertexArrayAttribEXT
+			,glVertexAttribFormat
+			,glVertexAttribIFormat
+			,glVertexAttribLFormat
+			,glVertexArrayAttribFormat
+			,glVertexArrayAttribIFormat
+			,glVertexArrayAttribLFormat
+			,glVertexArrayVertexAttribFormatEXT
+			,glVertexArrayVertexAttribIFormatEXT
+			,glVertexArrayVertexAttribLFormatEXT
+			,glVertexArrayBindingDivisor
+			,glVertexArrayVertexBindingDivisorEXT
+			,glVertexBindingDivisor
+			,glProvokingVertex
+		);
+		IRR_SYSTEM_DECLARE_DYNAMIC_FUNCTION_CALLER_CLASS(GLdrawing, OpenGLFunctionLoader
+			,glPrimitiveRestartIndex
+			,glDrawArraysInstanced
+			,glDrawArraysInstancedBaseInstance
+			,glDrawElementsInstancedBaseVertex
+			,glDrawElementsInstancedBaseVertexBaseInstance
+			,glDrawTransformFeedback
+			,glDrawTransformFeedbackInstanced
+			,glDrawTransformFeedbackStream
+			,glDrawTransformFeedbackStreamInstanced
+			,glDrawArraysIndirect
+			,glDrawElementsIndirect
+			,glMultiDrawArraysIndirect
+			,glMultiDrawElementsIndirect
+			,glMultiDrawArraysIndirectCount
+			,glMultiDrawElementsIndirectCount
+			,glMultiDrawArraysIndirectCountARB
+			,glMultiDrawElementsIndirectCountARB
+		);
+		IRR_SYSTEM_DECLARE_DYNAMIC_FUNCTION_CALLER_CLASS(GLtransformFeedback, OpenGLFunctionLoader
+			,glCreateTransformFeedbacks
+			,glGenTransformFeedbacks
+			,glDeleteTransformFeedbacks
+			,glBindTransformFeedback
+			,glBeginTransformFeedback
+			,glPauseTransformFeedback
+			,glResumeTransformFeedback
+			,glEndTransformFeedback
+			,glTransformFeedbackBufferBase
+			,glTransformFeedbackBufferRange
+		);
+		IRR_SYSTEM_DECLARE_DYNAMIC_FUNCTION_CALLER_CLASS(GLquery, OpenGLFunctionLoader
+			,glCreateQueries
+			,glGenQueries
+			,glDeleteQueries
+			,glIsQuery
+			,glBeginQuery
+			,glEndQuery
+			,glBeginQueryIndexed
+			,glEndQueryIndexed
+			,glGetQueryiv
+			,glGetQueryObjectuiv
+			,glGetQueryObjectui64v
+			,glGetQueryBufferObjectuiv
+			,glGetQueryBufferObjectui64v
+			,glQueryCounter
+			,glBeginConditionalRender
+			,glEndConditionalRender
+		);
+		IRR_SYSTEM_DECLARE_DYNAMIC_FUNCTION_CALLER_CLASS(GLdebug, OpenGLFunctionLoader
+			glDebugMessageControl
+			glDebugMessageControlARB
+			glDebugMessageCallback
+			glDebugMessageCallbackARB
+		);
+		IRR_SYSTEM_DECLARE_DYNAMIC_FUNCTION_CALLER_CLASS(GLOS, OpenGLFunctionLoader
+#if defined(WGL_EXT_swap_control) && !defined(_IRR_COMPILE_WITH_SDL_DEVICE_)
+			,wglSwapIntervalEXT
+#endif
+#if defined(GLX_SGI_swap_control) && !defined(_IRR_COMPILE_WITH_SDL_DEVICE_)
+			,glXSwapIntervalSGI
+#endif
+#if defined(GLX_EXT_swap_control) && !defined(_IRR_COMPILE_WITH_SDL_DEVICE_)
+			,glXSwapIntervalEXT
+#endif
+#if defined(GLX_MESA_swap_control) && !defined(_IRR_COMPILE_WITH_SDL_DEVICE_)
+			,glXSwapIntervalMESA
+#endif
+		);
+		IRR_SYSTEM_DECLARE_DYNAMIC_FUNCTION_CALLER_CLASS(GLprogram, OpenGLFunctionLoader
+			,glEnablei
+			,glDisablei
```

Comment:
you also need to get entry points for "vanilla" OpenGL 1.4 functions that we use, such as `glEnable`.... oh and btw `Enable/Disable` are common opengl functions they handle everything